tests/end_to_end/test_send_message_from_existing_template.py

```diff
@@ -256,13 +256,6 @@ def handle_existing_template_case(page):
     # Check to make sure that we've arrived at the next page.
     page.wait_for_load_state("domcontentloaded")
 
-    preview_button = page.get_by_text("Preview")
```

Comment:
I am not super familiar with this current flow. Does this button not exist anymore or is the test being odd and not recognizing the button?

Reply:
If I'm understanding the changes here correctly, the template used for this page changed and therefore this is an intermediary step/page that is no longer needed for this flow.

However, I didn't think we were getting rid of the message preview step in the actual message send flow?  That's what this test is for, so I wonder if the changes in this PR have effects elsewhere in the full send message flow.

Reply:
Ah yes, makes sense. Thank you for pointing that out. I made an adjustment for the tour vs regular send flow @ccostino @alexjanousekGSA